### PR TITLE
configuring routing annotation details

### PR DESCRIPTION
**Pull request by ismail1432**
Before explains how routing is working I think we should explain how install annotations.

### CODE REVIEW

**Comment by ismail1432** [comment]
If a newbie comes on this page he will copy/paste the annotation on top of his function, refresh the bowser and he’ll get a huge error because we don’t tell/explain to him that he had to download annotation package before use them :-)

**Comment by sroze** [comment]
Thank you @ismail1432 for this first contribution, really appreciated 👍 

I think this definitely makes sense and would improve the documentation for newcomers.

File: routing.rst [hunk 1]
```diff
@@ -12,6 +12,21 @@ URL of a page from ``/blog`` to ``/news``? How many links would you need to
 hunt down and update to make the change? If you're using Symfony's router,
 the change is simple.
 
+Before using the annotations, be sure that you installed the annotations support :
```
Comment (sroze) [inline routing.rst]: I'd say "If you want to use the annotations, make sure you have installed the `annotations` recipe:"

File: routing.rst [hunk 2]
```diff
@@ -12,6 +12,21 @@ URL of a page from ``/blog`` to ``/news``? How many links would you need to
...
+
+.. code-block:: terminal
+
+    $ composer require annotations
+
+Now, comment-out the YAML route by adding the ``#`` character:
```
Comment (sroze) [inline routing.rst]: "To turn on the routing based on annotations, un-comment the following lines of your `config/routes.yaml` file:"

**Comment by ismail1432** [comment]
Hi @sroze,
Thank you for your message and your improvements.I had them

File: routing.rst [hunk 3]
```diff
@@ -12,6 +12,21 @@ URL of a page from ``/blog`` to ``/news``? How many links would you need to
 hunt down and update to make the change? If you're using Symfony's router,
 the change is simple.
 
+If you want to use the annotations, be sure that you have installed the ``annotations`` recipe :
+
+.. code-block:: terminal
+
+    $ composer require annotations
+
+ To turn on the routing based on annotations, un-comment the following lines of your config/routes.yaml file by adding the ``#`` character :
```
Comment (wouterj) [inline routing.rst]: The first space should be removed and please add a line break after the first word crossing the 72th character (so it reads nicely in the editor).

**Review by wouterj — APPROVED** [review]
Left one minor comment that we can fix during merging (so no need for you to do the work if you don't have time).

Thanks for your contribution! As said on Slack, I fully agree.

**Comment by ismail1432** [comment]
Hi @wouterj, To be honest I am not comfortable

**Comment by weaverryan** [comment]
Hey guys!

We definitely need this, but I think we already got it done in #8924, right? Btw, that PR purposely has no mention of uncommenting anything: that's not necessary anymore (a file containing the route import is added by the recipe - this was added about a month ago I think).

**Comment by wouterj** [comment]
Oh, I'm sorry @ismail1432. It indeed looks like Ryan's PR fixes this already. So I'm closing this one.

Please continue reporting/fixing doc issues if you find them. I would love to merge contributions from you! ;) (if you need any guidance/help with our PR/merge process, feel free to ping me or any other doc guy on Slack)

**Comment by ismail1432** [comment]
No worries guys,
Thanks for your job !
I’ll continue to help you 💪🏽

**Pull request closed without merging**